example/android/app/jni/Application.mk

```diff
@@ -4,8 +4,8 @@ APP_ABI := all
 APP_OPTIM := release
 APP_PLATFORM := android-8
 # GCC 4.8 Toolchain - requires NDK r9
-NDK_TOOLCHAIN_VERSION = 4.8
+NDK_TOOLCHAIN_VERSION = clang
```

Comment:
I'm actually not sure anymore. I think I was trying it out on both compilers for warnings and probably forgot to discard the change. Maybe it was because of the need for C++14 in the support lib. I don't see a reason why it shouldn't work with GCC